langgraph/src/utils.ts

```diff
@@ -89,3 +94,40 @@ export class RunnableCallable<I = unknown, O = unknown> extends Runnable<I, O> {
     return returnValue;
   }
 }
+
+export class DrawableGraph extends Graph {
+  override extend(graph: Graph, prefix: string = "") {
```

Comment:
I think this might not be necessary anymore (in python too), isn't this the same as the implementation in core now?

Reply:
You are correct, thought it was doing some different prefixing but no.